src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    @Test
+    void testConstructorAndGetters() {
```

Comment:
1) Please, follow the method name convention
2) I think we don't need to test getters/setters separately because it will be tested in other methods anyway + they don't contain any logic